esp/templates/users/userchecklist.html

```diff
@@ -14,6 +14,21 @@
 <br />
 <br />
 
+<script type="text/javascript">
+    function checkAll(checkId){
+        var inputs = document.getElementsByTagName("input");
```

Comment:
Just realized -- I think it's faster and simpler to do e.g. `document.querySelectorAll('input[type="checkbox"].class')` -- no need to iterate over all the inputs in the page.  (Alternately, we have jquery, it's `$j` rather than `$` for historical reasons, so you could use that instead.)